Sources/ComposableArchitecture/Reducer.swift

```diff
@@ -79,7 +79,7 @@ public struct Reducer<State, Action, Environment> {
   /// global state, action and environment. It accomplishes this by providing 3 transformations to
   /// the method:
   ///
-  /// * A writeable key path that can get/set a piece of local state from the global state
+  /// * A writeable key path that can get/set a piece of local state from the global state.
```

Comment:
Nice! Noticed another typo while reading 😅 
```suggestion
  /// * A writable key path that can get/set a piece of local state from the global state.
```